package.json

```diff
@@ -120,24 +120,33 @@
     ]
   },
   "scripts": {
-    "compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
+    "compile": "tsc -watch -p ./",
+    "vscode:prepublish": "tsc -p ./",
     "postinstall": "node ./node_modules/vscode/bin/install"
   },
   "devDependencies": {
     "chai": "^3.4.1",
     "del": "^2.2.0",
     "gulp": "^3.9.0",
     "gulp-istanbul": "^1.1.1",
-    "gulp-mocha": "^2.2.0",
+    "gulp-mocha": "^3.0.1",
     "gulp-tslint": "^3.2.0",
     "gulp-util": "^3.0.7",
     "mocha-junit-reporter": "^1.12.0",
     "should": "^8.1.1",
     "tslint": "^2.5.0",
-    "typescript": "^1.8.10",
-    "vscode": "^0.11.x",
+    "typescript": "^2.0.3",
+    "vscode": "^1.0.0",
     "vsts-task-lib": "^0.9.18",
-    "yargs": "^5.0.0"
+    "yargs": "^5.0.0",
+    "@types/applicationinsights": "^0.15.16",
```

Comment:
Probably should lock this at 0.15.16 since AI is also locked to 0.15.16.  There's a reason.